issue_drafts/issue_0.md

```diff
@@ -1,3 +1,38 @@
 # Issue 0: Installing Snakemake
+In order to walk through this learning module, we will create a new Python environment and install Snakemake and other required packages.
 
+Please make sure you have Anaconda installed on your computer before proceeding with the following steps.
 
+If you are working in Windows, you will need to open your Anaconda prompt to follow the steps below. If you are working in Linux or macOS, you can just open your command line.
+
+## Install using environment.yaml
+You can create a Conda environment for this tutorial with all the required packages in one step by running the following command:
+`
+conda env create -f environment.yaml
+`
+
+Note: You may have issues installing with the environment.yaml file. This could be due to the fact that Anaconda sometimes has trouble installing Snakemake. If this is the case, follow the instructions below for a manual installation.
```

Comment:
```suggestion
Note: You may have issues installing with the environment.yaml file. This could be due to the fact that Conda sometimes has trouble installing Snakemake. If you received any errors when creating the environment, skip down to the **Create environment and install packages manually** section.

If you did not receive any errors when creating the environment, test that the Conda environment was created successfully by activating it: `conda activate snakemake-tutorial`

If that worked, let's also test out your installation of Snakemake by running `snakemake --help`. You should see the documentation for Snakemake printed to your console.

If you are unable to activate your Conda environment, you will need to start over with the manual installation. Proceed to the **Create environment and install packages manually** section.

If you activated your Conda environment, but Snakemake failed to run successfully, delete any failed installations of Snakemake by running `conda remove snakemake`, and then pick up with **Step 3** in the **Create environment and install packages manually** section.
```